File: ajay/array1.c
#include<stdio.h>
	void main(){
		
		int end,search,flag=0;
		
		 printf("Enter the size of array:\n");
                scanf("%d",&end);
		
		printf("enter the element:\n");	
		int arr[end];
		for(int i=0;i<end;i++){
			scanf("%d",&arr[i]);
		}
		
		printf("Enter the searching number:\n");
		scanf("%d",&search);
		
		for(int i=0;i<end;i++){

			if(arr[i]==search){
			
			
			printf("%d is present in %d number",search,i);
		}
		}
	}
	

File: JAVABOOK/Naresh/Array/Array1.java
public class Array1 {

	static int arr[] = new int[10];

	public static void main(String[] args){

		System.out.println(arr);

		for(int i = 0 ; i < arr.length ; i++){

			arr[i] = i*i ;
		
		System.out.println(arr[i]);
		}
		System.out.println("=========================length===============================");
		System.out.println(arr.length);
	}
}
